ocs2_ros_interfaces/src/mpc/MPC_ROS_Interface.cpp

```diff
@@ -132,6 +153,16 @@ ocs2_msgs::msg::MPCFlattenedController MPC_ROS_Interface::createMpcPolicyMsg(con
   mpcPolicyMsg.data.reserve(N);
   mpcPolicyMsg.post_event_indices.clear();
   mpcPolicyMsg.post_event_indices.reserve(primalSolution.postEventIndices_.size());
+  mpcPolicyMsg.joint_names.clear();
+  mpcPolicyMsg.joint_names.reserve(joint_names.size());
+
+  // push back
+  if(joint_names.size()){
+    for(const std::string& jname : joint_names){
+      mpcPolicyMsg.joint_names.emplace_back(jname);
+    }
+  }
```

Comment:
Is mpcPolyMsg.joint_names not a vector? Because then you could just do `mpcPolicyMsg.joint_names = joint_names`.